src/logic/operate.js

```diff
@@ -2,31 +2,32 @@ import Big from 'big.js';
 
 export default function operate(numOne, numTwo, operation) {
   let total;
-  const numOneBig = new Big(numOne);
-  const numTwoBig = new Big(numTwo);
+  Big.PE = 10;
+  const numOneBig = new Big(parseFloat(numOne));
+  const numTwoBig = new Big(parseFloat(numTwo));
   switch (operation) {
     case '+':
-      total = numOneBig + numTwoBig;
+      total = numOneBig.plus(numTwoBig).toPrecision(5).toString();
       break;
 
     case '-':
-      total = numOneBig - numTwoBig;
+      total = numOneBig.minus(numTwoBig).toPrecision(5).toString();
       break;
 
     case 'X':
-      total = numOneBig * numTwoBig;
+      total = numOneBig.times(numTwoBig).toPrecision(5).toString();
       break;
 
     case '÷':
-      total = numOneBig / numTwoBig;
-      break;
-
-    case '%':
-      total = numOneBig * (numTwoBig / 100);
+      try {
+        total = numOneBig.div(numTwoBig).toPrecision(5).toString();
+      } catch (error) {
+        total = 'NaN';
+      }
       break;
```

Comment:
in case of this error (division by zero error), you can write `NaN` on screen instead of `'0'` as you did
(Please do this before you merge)